desktop/apps/auction_support/client/registration_form.coffee

```diff
@@ -91,6 +91,8 @@ module.exports = class RegistrationForm extends ErrorHandlingForm
               reject "Registration submission error: #{xhr.responseJSON?.message}"
     .then (bidder) =>
       analyticsHooks.trigger 'registration:success', bidder_id: bidder.id
+      @undelegateEvents()
```

Comment:
Is this okay in the case where someone is registering _only_, or if errors occur during registration?